frontend/src/app/offers/create-offer/create-offer.component.html

```diff
@@ -0,0 +1,300 @@
+<div *ngIf="isAdmin; else hasOrg">
+  <p>Administrator nie może tworzyć nowych ofert.</p>
+</div>
+<ng-template #hasOrg>
+  <ng-container *ngIf="!hasOrganization; else offerForm">
+  <p>Aby dodać ofertę musisz posiadać organizację.
+    <a href="{{ djangoRoot }}/organizations/create">Możesz ją założyć tutaj</a>
+  </p>
+  </ng-container>
+</ng-template>
+<ng-template #offerForm>
+    <ngb-alert *ngIf="error" type="danger">
+      Wypełnij brakujące pola.
+    </ngb-alert>
+<h2>{{ inEditMode ? "Edycja" : "Tworzenie" }} oferty:</h2>
+<form (ngSubmit)="onSubmit(offer)" [formGroup]="form">
+  <div class="form-group row">
+    <label for="title" class="col-sm-3 col-form-label">Tytuł oferty:</label>
+    <div class="col-sm-6">
+      <input 
+        type="text"
+        class="form-control"
+        placeholder="Tytuł oferty..." 
+        id="title"
+        formControlName ="title"
+        required>
+      <span class="form-text text-muted" 
+      *ngIf="isFormInputInvalid('title')">
+        To pole jest obowiązkowe.</span>
+    </div>
+    <div class="col-sm-3">Określ, jak oferta będzie nazywać się w systemie, staraj się być dokładnym.</div>
+  </div>
+<div class="form-group row">
+    <label for="location" class="col-sm-3 col-form-label">Miejsce odbywania wolontariatu:</label>
+    <div class="col-sm-6">
+      <input 
+        type="text"
+        class="form-control"
+        placeholder="Miejsce odbywania wolontariatu..." 
+        id="location" 
+        formControlName="location"
+        required>
+        <span class="form-text text-muted" 
+        *ngIf="isFormInputInvalid('location')">
+          To pole jest obowiązkowe.</span>
+    </div>
+    <div class="col-sm-3">Określ, w jakim miejscu odbywać się będzie wolontariat.</div>
+  </div>
+ <div class="form-group row">
+    <label for="organization" class="col-sm-3 col-form-label">Z ramienia organizacji:</label>
+    <div class="col-sm-6">
+      <select
+        class="form-control"
+        id="organization"
+        formControlName="organization"
+        required>
+        <option  *ngFor="let organization of user?.organizations" [ngValue]="organization">
```

Comment:
@radoslawroszkowiak Could you help me here? I remember you told me to change ngValue to value, but when I do this, then with offer I am sending organization as `[object object]`, so what do you think I should do here?

Reply:
I don't remember the issue.

If it's working, the leave it as it is ;)